baremaps-server/src/main/java/org/apache/baremaps/server/TileJSON/Field.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.baremaps.server.TileJSON;
```

Comment:
According to oracle the package names should be lower case.
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Reply:
Addressed in https://github.com/apache/incubator-baremaps/pull/658/commits/27ac21ae07f2125193b824c3200704ccce71aa7f 
I quickly check the configuration with spotless and it is not enforced by it. Sonar has a rule for it java:S120.

https://rules.sonarsource.com/java/tag/convention/RSPEC-120